$MirOS: ports/misc/mc/patches/patch-src_cons_handler_c,v 1.1 2005/03/18 15:50:27 tg Exp $
$OpenBSD: patch-src_cons_handler_c,v 1.1 2004/04/11 00:40:59 naddy Exp $
--- src/cons.handler.c.orig	Mon Sep 23 06:43:01 2002
+++ src/cons.handler.c	Fri Oct 15 17:49:34 2004
@@ -144,7 +144,7 @@ void handle_console (unsigned char actio
 	    if (tty_name) {
 		/* Exec the console save/restore handler */
 		mc_conssaver = concat_dir_and_file (LIBDIR, "cons.saver");
-		execl (mc_conssaver, "cons.saver", tty_name, NULL);
+		execl (mc_conssaver, "cons.saver", tty_name, (char *)NULL);
 	    }
 	    /* Console is not a tty or execl() failed */
 	    console_flag = 0;
